### DISCUSSION TITLE
Updates for CoinLedger Export

### DISCUSSION THREAD

**Discussion by mrlowe** [pwillworth/dfkreport#19] [Q&A] [answered]
Hello! I'm just wondering if there's any plan to update the export for CoinLedger. It appears their Type field may have changed since the last time this report was touched, so exports definitely won't import without a lot of find-and-replacing.

**Comment by pwillworth** [comment, marked as answer]
Was not aware they made major changes, I'll take a look, thanks for the heads up.

**Reply by mrlowe** [reply]
I'm not sure how fixable it is ... I guess this wasn't the case in the past (dunno, never used CoinLedger) but now if there are any unrecognized tokens in your import, you have to map them to one of the tokens CoinLedger knows about. I talked to support about this, and they said they might be willing to add all the DFK tokens, but even beyond DFKBLOATER you've still got LPs to deal with, and that seems like a big ask.

**Reply by pwillworth** [reply]
Hmmmm, yeah, all I can do is look at the type options they have and do my best to map what happens in DFK to those.   For CoinLedger they don't really have types that correlate or make sense for a lot of stuff.  I think people just have to decide what they want to remove from the CSV before importing.
